.github/workflows/rust.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --verbose --all --locked
+          args: --verbose --all --locked -- --test-threads 1
```

Comment:
Why are these changes required?

Reply:
Ahhh ... didn't see the PR comment when reviewing ... could we add this to the description of the PR?